wled00/FX.h

```diff
@@ -650,6 +650,8 @@ class WS2812FX {
     void handle_palette(void);
 
     bool
+      firstRun = true,
```

Comment:
What does this do? Doesn't seem like its value is used anywhere.

Reply:
I had a `firstRun` variable setup because I thought that my code running on boot was causing a weird flash as I described on Discord [here](https://discordapp.com/channels/473448917040758787/718943978636050542/764759086843428874). It's not needed anymore and has been removed in the latest commit.